diff -ur node_modules/web3-eth/types/index.d.ts backup/node_modules/web3-eth/types/index.d.ts
--- node_modules/web3-eth/types/index.d.ts	2023-03-23 09:31:35.200862719 +0000
+++ backup/node_modules/web3-eth/types/index.d.ts	2023-03-23 07:50:31.540825400 +0000
@@ -60,7 +60,7 @@
     chain
 } from 'web3-core';
 
-export default class Eth {
+export class Eth {
     constructor();
     constructor(provider: provider);
     constructor(provider: provider, net: net.Socket);
diff -ur node_modules/web3-eth-ens/types/index.d.ts backup/node_modules/web3-eth-ens/types/index.d.ts
--- node_modules/web3-eth-ens/types/index.d.ts	2023-03-23 09:55:13.380871459 +0000
+++ backup/node_modules/web3-eth-ens/types/index.d.ts	2023-03-23 09:52:55.950870612 +0000
@@ -19,7 +19,7 @@
 
 import { PromiEvent, TransactionConfig, TransactionReceipt } from 'web3-core';
 import { TransactionRevertInstructionError } from 'web3-core-helpers';
-import Eth from 'web3-eth';
+import  Eth  from 'web3-eth';
 import { Contract } from 'web3-eth-contract';
 
 export interface ContentHash {
